app/javascript/components/PlaylistRamp.jsx

```diff
@@ -164,7 +203,27 @@ const Ramp = ({
               </div>
             </Col>
           </Row>
-          <div dangerouslySetInnerHTML={{ __html: comment_tag.content }} />
+          <div>
+            {comment && (
+              <div>
+                <h4>{comment_label}</h4>
+                <div>
+                  <span dangerouslySetInnerHTML={{ __html: description }} />
+                  {words.length > wordCount && (
+                    <a className="btn-link" style={expandBtn} onClick={handleClick}>
```

Comment:
Also needs to be updated here:
```suggestion
                    <a className="btn-link" style={expandBtn} onClick={handleDescriptionMoreLessClick}>
```